compiler/rustc_attr/src/builtin.rs

```diff
@@ -436,7 +436,12 @@ pub fn find_crate_name(sess: &Session, attrs: &[Attribute]) -> Option<Symbol> {
 }
 
 /// Tests if a cfg-pattern matches the cfg set
-pub fn cfg_matches(cfg: &ast::MetaItem, sess: &ParseSess, features: Option<&Features>) -> bool {
+pub fn cfg_matches(
+    cfg: &ast::MetaItem,
+    sess: &ParseSess,
+    node_id: Option<NodeId>,
```

Comment:
```suggestion
    lint_node_id: Option<NodeId>,
```
And maybe it's better to get rid of the `Option` too, and just pass `CRATE_NODE_ID` when necessary, not sure.

Reply:
Okay, will do it.